### YamlMime:ManagedReference
items:
- uid: System.Web.UI.AsyncPostBackErrorEventArgs
  id: AsyncPostBackErrorEventArgs
  children:
  - System.Web.UI.AsyncPostBackErrorEventArgs.#ctor(System.Exception)
  - System.Web.UI.AsyncPostBackErrorEventArgs.Exception
  langs:
  - csharp
  name: AsyncPostBackErrorEventArgs
  nameWithType: AsyncPostBackErrorEventArgs
  fullName: System.Web.UI.AsyncPostBackErrorEventArgs
  type: Class
  summary: "提供資料給<xref href=&quot;System.Web.UI.ScriptManager.AsyncPostBackError&quot;></xref>事件。"
  remarks: "AsyncPostBackErrorEventArgs 物件用來傳遞事件資料時<xref:System.Web.UI.ScriptManager.AsyncPostBackError>局部網頁呈現期間，會引發。</xref:System.Web.UI.ScriptManager.AsyncPostBackError> 事件資料即伺服器例外狀況的相關資訊。 使用<xref:System.Web.UI.AsyncPostBackErrorEventArgs.Exception%2A>屬性來存取事件資料。</xref:System.Web.UI.AsyncPostBackErrorEventArgs.Exception%2A>"
  syntax:
    content: 'public class AsyncPostBackErrorEventArgs : EventArgs'
  inheritance:
  - System.EventArgs
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Web.UI.AsyncPostBackErrorEventArgs.#ctor(System.Exception)
  id: '#ctor(System.Exception)'
  parent: System.Web.UI.AsyncPostBackErrorEventArgs
  langs:
  - csharp
  name: AsyncPostBackErrorEventArgs(Exception)
  nameWithType: AsyncPostBackErrorEventArgs.AsyncPostBackErrorEventArgs(Exception)
  fullName: System.Web.UI.AsyncPostBackErrorEventArgs.AsyncPostBackErrorEventArgs(Exception)
  type: Constructor
  assemblies:
  - System.Web.Extensions
  namespace: System.Web.UI
  summary: "初始化的新執行個體<xref href=&quot;System.Web.UI.AsyncPostBackErrorEventArgs&quot;></xref>類別。"
  syntax:
    content: public AsyncPostBackErrorEventArgs (Exception exception);
    parameters:
    - id: exception
      type: System.Exception
      description: "在局部網頁呈現期間引發的伺服器例外狀況。"
  overload: System.Web.UI.AsyncPostBackErrorEventArgs.#ctor*
  exceptions:
  - type: System.ArgumentException
    commentId: T:System.ArgumentException
    description: "<code>exception</code>is <xref uid=&quot;langword_csharp_null&quot; name=&quot;null&quot; href=&quot;&quot;></xref>."
  platform:
  - net462
- uid: System.Web.UI.AsyncPostBackErrorEventArgs.Exception
  id: Exception
  parent: System.Web.UI.AsyncPostBackErrorEventArgs
  langs:
  - csharp
  name: Exception
  nameWithType: AsyncPostBackErrorEventArgs.Exception
  fullName: System.Web.UI.AsyncPostBackErrorEventArgs.Exception
  type: Property
  assemblies:
  - System.Web.Extensions
  namespace: System.Web.UI
  summary: "取得<xref:System.Exception>局部網頁呈現期間引發。</xref:System.Exception>"
  syntax:
    content: public Exception Exception { get; }
    return:
      type: System.Exception
      description: "<xref:System.Exception>物件。</xref:System.Exception>"
  overload: System.Web.UI.AsyncPostBackErrorEventArgs.Exception*
  exceptions: []
  platform:
  - net462
references:
- uid: System.EventArgs
  isExternal: false
  name: System.EventArgs
- uid: System.ArgumentException
  isExternal: true
  name: System.ArgumentException
- uid: System.Web.UI.AsyncPostBackErrorEventArgs.#ctor(System.Exception)
  parent: System.Web.UI.AsyncPostBackErrorEventArgs
  isExternal: false
  name: AsyncPostBackErrorEventArgs(Exception)
  nameWithType: AsyncPostBackErrorEventArgs.AsyncPostBackErrorEventArgs(Exception)
  fullName: System.Web.UI.AsyncPostBackErrorEventArgs.AsyncPostBackErrorEventArgs(Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Web.UI.AsyncPostBackErrorEventArgs.Exception
  parent: System.Web.UI.AsyncPostBackErrorEventArgs
  isExternal: false
  name: Exception
  nameWithType: AsyncPostBackErrorEventArgs.Exception
  fullName: System.Web.UI.AsyncPostBackErrorEventArgs.Exception
- uid: System.Web.UI.AsyncPostBackErrorEventArgs.#ctor*
  parent: System.Web.UI.AsyncPostBackErrorEventArgs
  isExternal: false
  name: AsyncPostBackErrorEventArgs
  nameWithType: AsyncPostBackErrorEventArgs.AsyncPostBackErrorEventArgs
- uid: System.Web.UI.AsyncPostBackErrorEventArgs.Exception*
  parent: System.Web.UI.AsyncPostBackErrorEventArgs
  isExternal: false
  name: Exception
  nameWithType: AsyncPostBackErrorEventArgs.Exception
